apicast/src/policy.lua

```diff
@@ -0,0 +1,42 @@
+local _M = {
+    PHASES = {
```

Comment:
Should this table be exposed? Users of this module only call `.phases()`.

Reply:
True, no need for it.